src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tH = thead.querySelectorAll('th');
+
+const tbody = document.querySelector('tbody');
+const tBodyRows = [...tbody.rows];
+
+function usdNum(str) {
+  return Number(str.replace(/\D/g, ''));
+}
+
+thead.addEventListener('click', (ev) => {
+  const sortByIndex = [...tH].indexOf(ev.target);
+
+  tBodyRows.sort((a, b) => {
+    const aIn = a.cells[sortByIndex].innerText;
+    const bIn = b.cells[sortByIndex].innerText;
+
+    return usdNum(aIn) - usdNum(bIn) || aIn.localeCompare(bIn);
```

Comment:
let's rewrite it with `if` statement, cause if `usdNum(aIn) - usdNum(bIn)` return -1 js would check `aIn.localeCompare(bIn)` and it's incorrect